calcom/cal.com
# In-app users admin

## What does this PR do?

- Adds a new enterprise-only feature for ADMINs
- Allows ADMIN users to manage their instance users
- Adds UI to add users
- Adds UI to edit basic user info
- Adds UI to list users and delete from there

## Type of change



- New feature (non-breaking change which adds functionality)

## How should this be tested?



- [ ] As and ADMIN go to `/settings/admin/users`
- [ ] You should see your current users
- [ ] Try to add a new user
- [ ] Try to edit a current user
- [ ] Try to delete a current user

TODO in a follow-up

- Add search functionality user listing
- Add pagination for user listing
- Add more editable fields
  - metadata (for billing issues)
  - passwords?
  - Allow uploading/editing  avatars
- Add impersonation quick link?

Tagging @Jaibles and @keithwillcode for RFC

## Checklist



- I haven't performed a self-review of my own code and corrected any misspellings
- I haven't added tests that prove my fix is effective or that my feature works
